nemoguardrails/imports.py

```
@@ -106,10 +106,10 @@ def import_optional_dependency(
         module = importlib.import_module(name)
     except ImportError:
         if errors == "raise":
-            extra_msg = f" Install it via poetry install -E {extra}" if extra else ""
+            extra_msg = f" Install it via uv sync --extra {extra}" if extra else ""
             raise ImportError(f"Missing optional dependency '{install_name}'.{extra_msg}")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)